frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -893,7 +893,7 @@ export default function lineAreaBar(
 
   datas.map(data => {
     data.map(d => {
-      if (d._waterfallValue) {
+      if (typeof d._waterfallValue === "number") {
```

Comment:
```suggestion
      if (isFinite(d._waterfallValue)) {
```

It handles null as 0 and solves the issue Flamber found

Reply:
Shouldn't it be [Number.isFinite](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) instead?

Reply:
I stand corrected, we actually do *not* want `Number.isFinite` since it won't treat null as 0.

Reply:
Yes, maybe it is a bit implicit but works